### ISSUE TITLE
Android: Crash when click on Read Stories

### ISSUE THREAD

**Issue by shuvashish76** [samuelclay/NewsBlur#1549]
Crash logs:
```
FATAL EXCEPTION: main
Process: com.newsblur, PID: 3700
java.lang.NullPointerException: Attempt to invoke virtual method 'int android.view.View.getMeasuredHeight()' on a null object reference
	at com.newsblur.database.StoryViewAdapter.bindCommon(StoryViewAdapter.java:562)
	at com.newsblur.database.StoryViewAdapter.onBindViewHolder(StoryViewAdapter.java:517)
	at androidx.recyclerview.widget.RecyclerView$Adapter.onBindViewHolder(RecyclerView.java:7254)
	at androidx.recyclerview.widget.RecyclerView$Adapter.bindViewHolder(RecyclerView.java:7337)
	at androidx.recyclerview.widget.RecyclerView$Recycler.tryBindViewHolderByDeadline(RecyclerView.java:6194)
	at androidx.recyclerview.widget.RecyclerView$Recycler.tryGetViewHolderForPositionByDeadline(RecyclerView.java:6460)
	at androidx.recyclerview.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:6300)
	at androidx.recyclerview.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:6296)
	at androidx.recyclerview.widget.LinearLayoutManager$LayoutState.next(LinearLayoutManager.java:2330)
	at androidx.recyclerview.widget.GridLayoutManager.layoutChunk(GridLayoutManager.java:572)
	at androidx.recyclerview.widget.LinearLayoutManager.fill(LinearLayoutManager.java:1591)
	at androidx.recyclerview.widget.LinearLayoutManager.onLayoutChildren(LinearLayoutManager.java:668)
	at androidx.recyclerview.widget.GridLayoutManager.onLayoutChildren(GridLayoutManager.java:170)
	at androidx.recyclerview.widget.RecyclerView.dispatchLayoutStep2(RecyclerView.java:4309)
	at androidx.recyclerview.widget.RecyclerView.dispatchLayout(RecyclerView.java:4012)
	at androidx.recyclerview.widget.RecyclerView.onLayout(RecyclerView.java:4578)
	at android.view.View.layout(View.java:19663)
	at android.view.ViewGroup.layout(ViewGroup.java:6075)
	at android.widget.RelativeLayout.onLayout(RelativeLayout.java:1083)
	at android.view.View.layout(View.java:19663)
	at android.view.ViewGroup.layout(ViewGroup.java:6075)
	at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1791)
	at android.widget.LinearLayout.layoutHorizontal(LinearLayout.java:1780)
	at android.widget.LinearLayout.onLayout(LinearLayout.java:1546)
	at android.view.View.layout(View.java:19663)
	at android.view.ViewGroup.layout(ViewGroup.java:6075)
	at android.widget.RelativeLayout.onLayout(RelativeLayout.java:1083)
	at android.view.View.layout(View.java:19663)
	at android.view.ViewGroup.layout(ViewGroup.java:6075)
	at com.google.android.material.appbar.HeaderScrollingViewBehavior.layoutChild(HeaderScrollingViewBehavior.java:148)
	at com.google.android.material.appbar.ViewOffsetBehavior.onLayoutChild(ViewOffsetBehavior.java:43)
	at com.google.android.material.appbar.AppBarLayout$ScrollingViewBehavior.onLayoutChild(AppBarLayout.java:2003)
	at androidx.coordinatorlayout.widget.CoordinatorLayout.onLayout(CoordinatorLayout.java:918)
	at android.view.View.layout(View.java:19663)
	at android.view.ViewGroup.layout(ViewGroup.java:6075)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
	at android.view.View.layout(View.java:19663)
	at android.view.ViewGroup.layout(ViewGroup.java:6075)
	at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1791)
	at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1635)
	at android.widget.LinearLayout.onLayout(LinearLayout.java:1544)
	at android.view.View.layout(View.java:19663)
	at android.view.ViewGroup.layout(ViewGroup.java:6075)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
	at android.view.View.layout(View.java:19663)
	at android.view.ViewGroup.layout(ViewGroup.java:6075)
	at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1791)
	at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1635)
	at android.widget.LinearLayout.onLayout(LinearLayout.java:1544)
	at android.view.View.layout(View.java:19663)
	at android.view.ViewGroup.layout(ViewGroup.java:6075)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
	at com.android.internal.policy.DecorView.onLayout(DecorView.java:761)
	at android.view.View.layout(View.java:19663)
	at android.view.ViewGroup.layout(ViewGroup.java:6075)
	at android.view.ViewRootImpl.performLayout(ViewRootImpl.java:2500)
	at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2216)
	at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1396)
	at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:6773)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:966)
	at android.view.Choreographer.doCallbacks(Choreographer.java:778)
	at android.view.Choreographer.doFrame(Choreographer.java:713)
	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:952)
	at android.os.Handler.handleCallback(Handler.java:790)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:164)
	at android.app.ActivityThread.main(ActivityThread.java:6518)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
```
App version: 11.0 (193) (F-Droid)
Android version: 8.1.0
Model: Lenovo Tab4 8 plus (TB-8704X)

**Comment by sictiru** [comment]
@shuvashish76 The issue has been fixed in `11.0.1`. Please update and let us know if the problem persists.
https://github.com/samuelclay/NewsBlur/issues/1526

**Comment by shuvashish76** [comment]
No _11.0.1_ update on F-droid yet.
But checked with playstore version, no such issue there.

Thanks
